po/fr.po

```diff
@@ -230,12 +230,55 @@ msgstr "Modèle"
 msgid "Blueprint Components"
 msgstr "Composants du Modèle"
 
+#: build/messages/components/Modal/CreateBlueprint.json
+#. [components.Modal.2544705954]
+#. defaultMessage is:
+#. Blueprint names cannot contain spaces or the character: {invalidChar}
+msgid "Blueprint names cannot contain spaces or the character: {invalidChar}"
+msgstr ""
+"Le noms de topologie ne peuvent pas contenir d'espaces ou de caractère : "
+"{invalidChar}"
+
+#: build/messages/components/Modal/CreateBlueprint.json
+#. [components.Modal.1134732033]
+#. defaultMessage is:
+#. Blueprint names cannot contain spaces or the characters: {invalidChar}
+msgid "Blueprint names cannot contain spaces or the characters: {invalidChar}"
+msgstr ""
+"Le noms de topologie ne peuvent pas contenir d'espaces ou de caractères : "
+"{invalidChar}"
+
+#: build/messages/components/Modal/CreateBlueprint.json
+#. [components.Modal.2803906810]
+#. defaultMessage is:
+#. Blueprint names cannot contain spaces.
+msgid "Blueprint names cannot contain spaces."
+msgstr "Le noms de modèles (topologie) ne peuvent pas contenir d'espaces."
+
+#: build/messages/components/Modal/CreateBlueprint.json
+#. [components.Modal.2781854456]
+#. defaultMessage is:
+#. Blueprint names cannot contain the character: {invalidChar}
+msgid "Blueprint names cannot contain the character: {invalidChar}"
+msgstr ""
+"Le noms de modèles (topologie) ne peuvent pas contenir le caractère : "
+"{invalidChar}"
+
+#: build/messages/components/Modal/CreateBlueprint.json
+#. [components.Modal.3918401128]
+#. defaultMessage is:
+#. Blueprint names cannot contain the characters: {invalidChar}
+msgid "Blueprint names cannot contain the characters: {invalidChar}"
+msgstr ""
+"Le noms de modèles (topologie) ne peuvent pas contenir les caractères : "
+"{invalidChar}"
+
 #: build/messages/pages/blueprints/index.json
 #. [pages.blueprints.blueprintsTitle]
 #. defaultMessage is:
 #. Blueprints
 msgid "Blueprints"
-msgstr "Modèles"
+msgstr "Modèles "
```

Comment:
Interesting change. Could this have some reason or just typo that should be changed back in Zanata?

Reply:
Looks like a typo to me. The original string doesn't have it and there shouldn't be a space where this appears in the UI.

Nice catch!